Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -61,13 +62,21 @@ done
 num_vehicles=${NUM_VEHICLES:=3}
 export PX4_SIM_MODEL=${VEHICLE_MODEL:=iris}
 world=${WORLD:=empty}
+
+ARR_VEHICLE=(`echo $VEHICLE_MODEL | tr '_' ' '`)
+if [ ${arrModel[1]} == "rtps" ]
```

Comment:
I think arrModel is not being set anywhere

Reply:
OK. I will remove it and add "t" option instead.